src/documents/invoice/invoiceV3.ts

```diff
@@ -47,6 +58,23 @@ export class InvoiceV3 extends Document {
   /** The company registration information for the customer. */
   customerCompanyRegistration: CompanyRegistration[] = [];
 
+  /** The total amount without the tax value. */
```

Comment:
add a `@deprecated` tag to the get/set functions

https://jsdoc.app/tags-deprecated.html